administrator/components/com_menus/Service/HTML/Menus.php

```diff
@@ -260,4 +121,32 @@ public function visibility($params)
 
 		return ($show_menu === 0) ? '<span class="badge badge-secondary">' . Text::_('COM_MENUS_LABEL_HIDDEN') . '</span>' : '';
 	}
+
+	/**
+	 * Returns the enabled state of the component
+	 *
+	 * @param   string  $componentname  Component name of the item.
+	 *
+	 * @return  string  The Html code
+	 *
+	 * @since   4.0
+	 */
+	public function enabled($componentname)
+	{
+		if (!empty($componentname))
+		{
+			$db = Factory::getDbo();
+			$query = $db->getQuery(true);
+
+			$query->select($db->quoteName('a.enabled'))
+				->from($db->quoteName('#__extensions', 'a'))
+				->where($db->quoteName('a.element') . ' = ' . $db->quote($componentname));
```

Comment:
```suggestion
				->where($db->quoteName('a.element') . ' = :element')
				    ->bind(':element', $componentname);
```

all queries should be prepared statement